### DISCUSSION TITLE
Move to PHP v8

### DISCUSSION THREAD

**Discussion by figdot1** [lowag/libremaint#10] [General]
Just wondering if it is on a timeline? I have run Libremaint locally under PHP 7 but my webserver is fixed to PHP 8+. Happy to put in some effort if this is part of the general direction.

**Comment by lowag** [comment]
I tried with v8.3 and seemed to be OK. Have you experienced any issue with that?

**Reply by figdot1** [reply]
Thanks for the quick answer. I did not, but will now. Sorry for the question. My previous experience with PHP apps moving from V7 to V8 has been disappointing and so lowered my expectations.
